### DISCUSSION TITLE
versionFor() - that also searches for versions online

### DISCUSSION THREAD

**Discussion by jschneider** [Splitties/refreshVersions#715] [Q&A] [answered]
I am creating a gradle tasks that adds additional dependencies. 
Everything works fine so far. 

Now, I want to add the version number to the version.properties file. 
"gradle build" does this. But I would like to do this automatically in my task.

I am looking for something like: versionForAndLookupOnlineIfDoesNotExist().

Any ideas how I could solve this?

**Comment by LouisCAD** [comment, marked as answer]
Put a dependency notation with an underscore as the version, and you'll get what you're looking for.

For example, `versionFor(AndroidX.compose.compiler)` just works.
